simple/src/MediumProblem/StringMerge.java

```diff
@@ -0,0 +1,25 @@
+package MediumProblem;
+
+public class StringMerge {
+    public void merge(String nameOne, String nameTwo) {
+        int lengthNameOne = 0, lengthNameTwo = 0;
+
+        while (lengthNameOne < nameOne.length() || lengthNameTwo < nameTwo.length()) {
+            if (lengthNameOne < nameOne.length()) {
+                System.out.print(nameOne.charAt(lengthNameOne));
+                lengthNameOne++;
+            }
+            if (lengthNameTwo < nameTwo.charAt(lengthNameTwo)) {
+                System.out.print(nameTwo.charAt(lengthNameTwo));
+                lengthNameTwo++;
+            }
+        }
+    }
```

Comment:
you are just printing the char values here, create a String to merge two String and then print that String value

Reply:
done